app.py

```diff
@@ -15,9 +15,10 @@ def webhook():
     # We don't want to reply to ourselves!
     if random.random() < .05:
         msg = 'Dislike Button'
+        if msg != data.get('message'):  # Check if msg is not equal to the message received
         send_message(msg, data['id'])
```

Comment:
Is this valid Python?

Reply:
why wouldnt it be? did i do something wrong

Reply:
The line under the if isn't indented